tests/test_controller.py

```diff
@@ -856,14 +864,16 @@ def test_series(self):
         eq_(200, response.status_code)
         feed = feedparser.parse(response.data)
         eq_('Other Books in the Like As If Whatever Mysteries series', feed['feed']['title'])
```

Comment:
Since you now include a book in its own feed, 'Other Books in...' isn't apropriate anymore.